src/js/mixins/WindowListenerMixin.js

```diff
@@ -0,0 +1,20 @@
+import { each } from 'lodash';
+import $ from 'jQuery';
+
+const $window = $(window);
+
+export default {
+  componentDidMount() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).